### PR TITLE
[REVIEW]  Add docstring for merge function: Fixes #219

### LINKED ISSUE #219: Docstring needed for merge function
There's currently no docstring for the merge function and considering the parameters that we support differ from Pandas we should correct this ASAP. See: https://github.com/gpuopenanalytics/pygdf/blob/master/pygdf/dataframe.py#L657

### PR DESCRIPTION
**Pull request by andersy005**
Fixes https://github.com/gpuopenanalytics/pygdf/issues/219

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by andersy005** [comment]
cc @scopatz @kkraus14

File: pygdf/dataframe.py [hunk 1]
```diff
@@ -665,6 +665,42 @@ def _n_largest_or_smallest(self, method, n, columns, keep):
 
     def merge(self, other, on=None, how='left', lsuffix='_x', rsuffix='_y',
               type='sort'):
+        """Merge GPU DataFrame objects by performing a database-style join operation
+        by columns or indexes.
+
+        Parameters
+        ----------
+        other: DataFrame
```
Comment (scopatz) [inline pygdf/dataframe.py]: numpydoc has spaces on both sides of the `:`

**Comment by kkraus14** [comment]
add to whitelist

**Comment by kkraus14** [comment]
LGTM, no need to wait for tests to pass, thanks @andersy005!

**Pull request merged**